### PR TITLE
Style guide conformance

### PR DESCRIPTION
**Pull request by bpbond**
Code tweaks for PEcAn style. To prove I’m not *just* a troublemaker.

### CODE REVIEW

File: modules/allometry/R/AllomAve.R [hunk 1]
```diff
@@ -64,35 +64,37 @@ AllomAve <- function(pfts,components=6,outdir=NULL,con=NULL,field=NULL,
   require(coda)
   
   nested.range <- function(obs){
-    w = NULL
+    w <- NULL
     for(i in 1:length(obs)){
       for(j in 1:length(obs[[i]])){
-        w = t(apply(cbind(w,t(sapply(obs[[i]][[j]],range))),1,range))
+        w <- t(apply(cbind(w,t(sapply(obs[[i]][[j]],range))),1,range))
```
Comment (dlebauer) [inline modules/allometry/R/AllomAve.R]: Great to have your input. Since this is about style

* spaces after `,`

**Comment by bpbond** [comment]
Doh! Hard to tell from the Travis log if this is something I did or not. Don't want to waste your time @mdietze - sorry.

File: modules/allometry/R/AllomAve.R [hunk 2]
```diff
@@ -64,35 +64,37 @@ AllomAve <- function(pfts,components=6,outdir=NULL,con=NULL,field=NULL,
...
       }
     }
     return(t(w))
   }
     
-  if(is.null(outdir)) outdir = getwd()
+  if(is.null(outdir)){
+  	outdir <- getwd()
+  }
```
Comment (dlebauer) [inline modules/allometry/R/AllomAve.R]: 👊🏼

File: modules/allometry/R/AllomAve.R (hunk 1 above)
Comment (bpbond) [inline modules/allometry/R/AllomAve.R]: That totally bugs me too. Happy to change.

File: modules/allometry/R/AllomAve.R [hunk 3]
```diff
@@ -106,39 +108,39 @@ AllomAve <- function(pfts,components=6,outdir=NULL,con=NULL,field=NULL,
       obs <- list()
       for(i in 1:nchain){
         if(component == 40){
-          allom.out = allom.BayesFit(allom,ngibbs,"exp",dmin,dmax)
+          allom.out <- allom.BayesFit(allom,ngibbs,"exp",dmin,dmax)
         } else {
-          allom.out = allom.BayesFit(allom,ngibbs,dmin=dmin,dmax=dmax)
+          allom.out <- allom.BayesFit(allom,ngibbs,dmin=dmin,dmax=dmax)
```
Comment (dlebauer) [inline modules/allometry/R/AllomAve.R]: spaces around `=`

**Comment by dlebauer** [comment]
Probably would be a good idea to use the `formatr` package on any new files (if not the whole repo in one swoop)

**Comment by dlebauer** [comment]
@bpbond this is an error in an unrelated package (PEcAn.DB) perhaps related to yesterday's release of BETYdb but almost certainly not one of these changes.

**Comment by bpbond** [comment]
@dlebauer now that is a good idea.

File: modules/allometry/R/AllomAve.R (hunk 2 above)
Comment (bpbond) [inline modules/allometry/R/AllomAve.R]: Did I break this? Not seeing anything. May need more coffee.

**Comment by mdietze** [comment]
@bpbond I've let this PR languish too long. I didn't pull it immediately because, unbeknownst to you, I have a dev fork where I've made a bunch of changes to the allom code. But I haven't found the time to deal with that module / paper for months, and now there are conflicts with changes Andria made. So I'm going to close the PR, merge in your changes to resolve the conflict, and the push it back in. Sorry for letting this drag out.

**Comment by bpbond** [comment]
No worries, and thanks. Happy to submit other PRs in this vein (code clarity and style) if useful.

**Comment by mdietze** [comment]
Happy to have them! Also happy to have any PR that includes new content (analyses, models, data ingest, visualization, etc).

**Pull request closed without merging**